### PR TITLE
add 9 functions

### PR DESCRIPTION
**Pull request by kamesa7**
added
norm() // length of the vector
unit() // the vector of length 1
scale(scalar)
3 distanceTo(other)
innerProduct(other)
manhattanDistanceTo(other)
toArray()

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -119,6 +119,47 @@ Vec3.prototype.min = function(other) {
...
   return new Vec3(Math.max(this.x, other.x), Math.max(this.y, other.y), Math.max(this.z, other.z));
 };
+Vec3.prototype.norm = function() {
+  return Math.sqrt(this.x * this.x + this.y * this.y + this.z * this.z);
+};
+Vec3.prototype.unit = function() {
+  var norm = this.norm();
+  if(norm === 0) {
+    return this;
+  } else {
```
Comment (rom1504) [inline index.js:129]: Would be better to return a new vector here to he coherent with the other case (callers should be able to assume mutability or immutability)

**Review by rom1504 — CHANGES_REQUESTED** [review]
One small change to make, other than that it looks good to me.

**Comment by kamesa7** [comment]
Thank you for the suggestion. I fixed that point.

**Review by rom1504 — APPROVED** [review]
LGTM
@andrewrk can you merge ? (Maybe you could add me as maintainer for this lib too as it's mainly used in mineflayer ?)

**Comment by rom1504** [comment]
Why did you close ?

**Comment by andrewrk** [comment]
I will be happy to add @rom1504 as a maintainer when I get back home

**Comment by andrewrk** [comment]
@rom1504 you now have write access to this repository and you are an owner of the npm package.

**Comment by rom1504** [comment]
Thanks !

**Pull request merged**